### ISSUE TITLE
Connect worker via RPN to download.kiwix.org

### ISSUE THREAD

**Issue by kelson42** [kiwix/cardshop#252]
This should definitly speed-up download of data because:
* no competition on bandwith
* More bandwidth in the rpn than on the public interface

**Comment by rgaudin** [comment]
Absolutely, although this task is mostly about setting up this network on scaleway and configuring the mirror server to answer on that second IP, probably using another dns entry.

On hotspot side, support for specifying mirror from the command line and rewriting catalog url to use that mirror.

On cardshop itself, supporting mirror change via env and passing down that to the creator's command line.

**Comment by kelson42** [comment]
We need a new download.kiwix.org server allowing RPN.

**Comment by kelson42** [comment]
@rgaudin I guess we can close the ticket now?

**Comment by rgaudin** [comment]
What was done:
- configured RPNv1 in Scaleway webui (demo supports v2 but download only supports v1)
- enable those second iface in both machines (see bellow)
- ensured both machines could reach each-other
- added hosts in both machines (to ease maintenance)
- cardshop creator adds a new host to its container: `docker run --add-host download.kiwix.org:10.91.56.88 ...`

```
10.91.56.88	rpn-download
10.91.233.86	rpn-demo
```

## demo (Ubuntu 18.04)
Edited `/etc/netplan/01-netcfg.yaml`

```yaml
network:
  version: 2
  renderer: networkd
  ethernets:
    enp1s0f0:
      dhcp4: yes
    enp1s0f1:
      dhcp4: yes
```

```sh
netplan try
```

## download (Debian 9)

Edited `/etc/network/interfaces`

```
# The loopback network interface
auto lo
iface lo inet loopback

# The primary network interface
allow-hotplug eno1
iface eno1 inet dhcp

allow-hotplug eno2
iface eno2 inet dhcp
```

```sh
ifconfig ino2 up
```
